libraries/mapping/carma-map-layers/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
+  "extends": "../../../tsconfig.legacy.base.json",
```

Comment:
@d4v3000  
use this as baseline if you have a lot of errors:
 "extends": "../../../tsconfig.legacy.base.json",
 if possible use 
 tsconfig.base.json
 or even
 tsconfig.strict.base.json